packages/create-pantheon-decoupled-kit/src/templates/next-wp/pages/index.jsx

```diff
@@ -0,0 +1,72 @@
+import { NextSeo } from 'next-seo';
+import { setOutgoingHeaders } from '../lib/setOutgoingHeaders';
+
+import Image from 'next/image';
+import Layout from '../components/layout';
+
+import { PostGridItem } from '../components/grid';
+import { withGrid } from '@pantheon-systems/nextjs-kit';
+
+import { getFooterMenu } from '../lib/Menus';
+import { getLatestPosts } from '../lib/Posts';
+
+export default function Home({ menuItems, posts }) {
+	const HomepageHeader = () => (
+		<div className="prose sm:prose-xl mt-20 flex flex-col mx-auto max-w-fit">
+			<h1 className="prose text-4xl text-center h-full">
+				Welcome to{' '}
+				<a
+					className="text-blue-600 no-underline hover:underline"
+					href="https://nextjs.org"
+				>
+					Next.js!
+				</a>
+			</h1>
+
+			<div className="text-2xl">
+				<div className="bg-black text-white rounded flex items-center justify-center p-4">
+					Decoupled WordPress on{' '}
+					<Image
+						src="/pantheon.png"
```

Comment:
Not sure why, but when I create a new starter using the CLI the Pantheon logo displays as broken.

Reply:
Ah yes, forgot about the cases where binary files need to be copied directly instead of reading the contents and writing it back. Should be fixed now!